syntax = "proto3";
package streetgangpb;


message GetVersionResponse
{
	string version = 1;
}
